cg/store/api/update.py

```diff
@@ -0,0 +1,16 @@
+"""Module to updating existing records in status-db tables"""
+
```

Comment:
Do you think this can be moved into UploadVogueAPI? 
This will be used only in the context of uploading to vogue, and should be used after an analysis has been uploaded